doc/lightning-listfunds.7.txt

```diff
@@ -0,0 +1,63 @@
+LIGHTNING-LISTFUNDS(7)
+======================
+:doctype: manpage
+
+NAME
+----
+lightning-listfunds - Protocol showing all funds currently managed by 
```

Comment:
This is a command, not a protocol.

Reply:
Existing manpages use this term, but I am uncertain, how to differentiate "command" and "protocol".

Reply:
Must have missed that until now, sorry. Being an RPC we can call it procedure, but protocol is definitely wrong.

Reply:
If it helps I also pushed ```dd06cdd``` with the same replacement in the rest of the pages